terraform-plans/configs/charm-nginx_main.tfvars

```diff
@@ -7,4 +7,32 @@ templates = {
     destination = ".github/CODEOWNERS"
     vars        = {}
   }
+  check = {
+    source      = "./templates/github/charm_check.yaml.tftpl"
+    destination = ".github/workflows/check.yaml"
+    vars        = {
+      # github hosted runners are amd64
+      # Ubuntu_ARM64_4C_16G_01 is the github-hosted arm64 runner we have access to.
+      # We prefer the github runners because they are smaller machines and save resources.
+      # If we have issues with it, we can switch to the larger and more numerous self-hosted options:
+      # - runs-on: [self-hosted, jammy, ARM64]
+      runs_on = "[[ubuntu-22.04], [Ubuntu_ARM64_4C_16G_01]]",
```

Comment:
@aieri we can't add arm64 runners here, because the nginx charm doesn't have support for arm64

Reply:
duh of course... updating